### ISSUE TITLE
All XrdEc tests fail on Linux SPARC systems due to alignment issues (bus error)

### ISSUE THREAD

**Issue by glaubitz** [xrootd/xrootd#2131]
Running the testsuite on Linux SPARC fails with the following tests failing:

```
The following tests FAILED:
         20 - XrdEc::AlignedWriteTest (Bus error)
         21 - XrdEc::SmallWriteTest (Bus error)
         22 - XrdEc::BigWriteTest (Bus error)
         23 - XrdEc::VectorReadTest (Bus error)
         24 - XrdEc::IllegalVectorReadTest (Bus error)
         25 - XrdEc::AlignedWrite1MissingTest (Bus error)
         26 - XrdEc::AlignedWrite2MissingTest (Bus error)
         27 - XrdEc::AlignedWriteTestIsalCrcNoMt (Bus error)
         28 - XrdEc::SmallWriteTestIsalCrcNoMt (Bus error)
         29 - XrdEc::BigWriteTestIsalCrcNoMt (Bus error)
         30 - XrdEc::AlignedWrite1MissingTestIsalCrcNoMt (Bus error)
         31 - XrdEc::AlignedWrite2MissingTestIsalCrcNoMt (Bus error)
```

Obtaining a backtrace with gdb is not possible as the stack gets corrupted.

SSH access to SPARC machines for testing can be obtained through the GCC Compile Farm: https://gcc.gnu.org/wiki/CompileFarm

**Comment by amadio** [comment]
Hi, are you using XRootD on a SPARC machine? We unfortunately do not have enough resources to support these less used platforms, but we will gladly review and merge patches fixing these issues if they come in a pull request.

**Comment by amadio** [comment]
Fixed by #2144.

**Comment by amadio** [comment]
@glaubitz Thanks for the tip about the GCC compile farm. In the end, however, I could not install xrootd dependencies there to test, so what I did was to add a QEMU-based workflow that only runs on demand to test alternative architectures. You can see an example run here (and can run on your fork if you have one as well): https://github.com/xrootd/xrootd/actions/runs/7171402996/job/19526316751
